book/src/tutorial/app.md

```diff
@@ -16,7 +16,23 @@ cd my-uefi-app
 Add a few dependencies:
 
 ```sh
-cargo add log uefi
+cargo add log
+cargo add uefi --features logger panic_handler
+```
+
+Also you have to add 
+```toml
+[profile.dev]
+panic="abort"
```

Comment:
This shouldn't normally be needed, what error did you run into without `panic="abort"`?

Reply:
ahh, I think this was from when I was trying to compile with the wrong target. I'll remove it.